README.md

```diff
@@ -2,11 +2,15 @@
 
 Skupper enables cloud communication by enabling you to create a Virtual Application Network.
 
-This upper layer network decouples addressing from the underlying network infrastructure, providing a secure and simple alternative to VPNs.
+This application layer network decouples addressing from the underlying network infrastructure.
+One example usage is to provide a secure and simple alternative to a VPN.
 
 You can use Skupper to create a network from namespaces inn one or more Kubernetes clusters as described in the [Getting Started](https://skupper.io/start/index.html).
-This guide describes a simple network, however there are no restrictions on the topology created which can include redundant paths. 
-You can also create more complex (full mesh) networks, Skupper then supports anycast and multicast communication.
+This guide describes a simple network, however there are no restrictions on the topology created which can include redundant paths.
+
+You can also create more complex networks, for example, a full mesh network.
```

Comment:
I don't think a full-mesh is a good example of a complex network. The previous sentence I think is perhaps enough. Maybe we could also explicitly state that sites don't have to be directly connected to be able to communicate, as long as they are connected to the network in some way.